src/utils/logic-app/designerUtils.ts

```diff
@@ -21,7 +21,7 @@ interface IGetWebviewContentOptions {
     workflowId: string;
 }
 
-const version = "1.31126.1.513093.181130-1946";
+const version = "1.31203.1.28076.181205-1625";
```

Comment:
If the version number is going to be the same in both spots, perhaps put it in a settings/constants file? Make it a single spot to change?